packages/web-react/src/components/FieldGroup/README.md

```diff
@@ -0,0 +1,124 @@
+# FieldGroup
+
+FieldGroup is a component that groups form fields together. Additionally, it provides a common label, helper text, and
```

Comment:
Can we have every sentence on a new line, please? For better future diffs ;-)

Reply:
done

Reply:
This was meant for the entire document. But there is a topic for this on our thursday meeting 🥌